src/redux/walletconnect.ts

```diff
@@ -397,7 +397,7 @@ export const walletConnectOnSessionRequest = (
             error,
             payload,
           });
-          throw error;
+          return;
```

Comment:
Already caught, but no need to bubble this to the catch boundary since we've got all the context we need in the log lines above.